### PR TITLE
Document format correction

### PR DESCRIPTION
**Pull request by qy3u**
Minor amendments to the document.

r? @steveklabnik

### CODE REVIEW

File: src/libstd/sys/unix/ext/fs.rs [hunk 1]
```diff
@@ -262,7 +263,8 @@ pub trait PermissionsExt {
     ///
     ///     permissions.set_mode(0o644); // Read/write for owner and read for others.
     ///     assert_eq!(permissions.mode(), 0o644);
-    ///     Ok(()) }
+    ///     Ok(())
```
Comment (jonas-schievink) [inline src/libstd/sys/unix/ext/fs.rs:266]: Please remove the trailing whitespace:
```suggestion
    ///     Ok(())
```

**Review by jonas-schievink — COMMENTED** [review]
Thanks!

File: src/libstd/sys/unix/ext/fs.rs (hunk 1 above)
Comment (qy3u) [reply src/libstd/sys/unix/ext/fs.rs]: updated

**Review by steveklabnik — APPROVED** [review]

**Pull request merged**